arctic/chunkstore/chunkstore.py

```diff
@@ -429,9 +429,86 @@ def update(self, symbol, item, chunk_range=None, upsert=False, **kwargs):
             self.__update(sym, item, combine_method=self.__replace, chunk_range=chunk_range)
 
     def get_info(self, symbol):
+        """
+        Returns information about the symbol, in a dictionary
+
+        Parameters
+        ----------
+        symbol: str
+            the symbol for the given item in the DB
+        """
         sym = self._get_symbol_info(symbol)
+        if not sym:
+            raise NoDataFoundException("Symbol does not exist.")
         ret = {}
         ret['chunk_count'] = sym[CHUNK_COUNT]
         ret['len'] = sym[LEN]
         ret['metadata'] = sym[METADATA]
         return ret
+
+    def get_chunk_ranges(self, symbol, chunk_range=None, reverse=False):
+        """
+        Returns a generator of (Start, End) tuples for each chunk in the symbol
+
+        Parameters
+        ----------
+        symbol: str
+            the symbol for the given item in the DB
+        chunk_range: None, or a range object
+            allows you to subset the chunks by range
+        reverse: boolean
+            return the chunk ranges in reverse order
+        """
+        sym = self._get_symbol_info(symbol)
+        if not sym:
+            raise NoDataFoundException("Symbol does not exist.")
+        c = CHUNKER_MAP[sym[CHUNKER]]
+
+        spec = {SYMBOL: symbol}
+        if chunk_range:
+            spec.update(CHUNKER_MAP[sym[CHUNKER]].to_mongo(chunk_range))
+
+        for x in self._collection.find(spec, projection=[START, END], sort=[(START, pymongo.ASCENDING if not reverse else pymongo.DESCENDING)]):
+            yield (c.chunk_to_str(x[START]), c.chunk_to_str(x[END]))
+
+    def start_iterator(self, symbol, chunk_range=None):
```

Comment:
maybe `iterator` and `reverse_iterator`  - to match Python and C++ naming?  Alternatively, given this just calls `read` could make it parameters on the read method?